<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helloLayoutSubtab</class>
 <widget class="QWidget" name="helloLayoutSubtab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;These layout options control the positioning of barbs. The barbs can be regularly spaced in the x, y, and z coordinates in a 3-dimensional region. The horizontal spacing of barbs can instead be aligned to the X,Y coordinate variables by checking &amp;quot;Align to XY grid&amp;quot;. The barbs can be positioned vertically a fixed distance above the terrain if there is a height variable.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Start Point:  X:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="xCoord1Edit">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Specify the X, Y, and Z dimensions of the array of barbs</string>
         </property>
         <property name="whatsThis">
          <string>The Dimensions indicate the x, y, and z grid dimensions of the barbs.  When the barbs are aligned to the XY grid, the X and Y dimensions indicate the horizontal size of the data-aligned grid on which the barbs are situated.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Y:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="yCoord1Edit">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Specify the X, Y, and Z dimensions of the array of barbs</string>
         </property>
         <property name="whatsThis">
          <string>The Dimensions indicate the x, y, and z grid dimensions of the barbs.  When the barbs are aligned to the XY grid, the X and Y dimensions indicate the horizontal size of the data-aligned grid on which the barbs are situated.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Z:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="zCoord1Edit">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Specify the X, Y, and Z dimensions of the array of barbs</string>
         </property>
         <property name="whatsThis">
          <string>The Dimensions indicate the x, y, and z grid dimensions of the barbs.  When the barbs are aligned to the XY grid, the X and Y dimensions indicate the horizontal size of the data-aligned grid on which the barbs are situated.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>End Point:  X:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="xCoord2Edit">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Specify the X, Y, and Z dimensions of the array of barbs</string>
         </property>
         <property name="whatsThis">
          <string>The Dimensions indicate the x, y, and z grid dimensions of the barbs.  When the barbs are aligned to the XY grid, the X and Y dimensions indicate the horizontal size of the data-aligned grid on which the barbs are situated.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Y:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="yCoord2Edit">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Specify the X, Y, and Z dimensions of the array of barbs</string>
         </property>
         <property name="whatsThis">
          <string>The Dimensions indicate the x, y, and z grid dimensions of the barbs.  When the barbs are aligned to the XY grid, the X and Y dimensions indicate the horizontal size of the data-aligned grid on which the barbs are situated.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Z:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="zCoord2Edit">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Specify the X, Y, and Z dimensions of the array of barbs</string>
         </property>
         <property name="whatsThis">
          <string>The Dimensions indicate the x, y, and z grid dimensions of the barbs.  When the barbs are aligned to the XY grid, the X and Y dimensions indicate the horizontal size of the data-aligned grid on which the barbs are situated.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <includes>
  <include location="local">BoxSliderFrame.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
